tests/containers/bci_prepare.pm

```diff
@@ -139,6 +138,11 @@ sub run {
 
     return if (get_var('HELM_CONFIG') && !($host_distri == "sles" && $version == 15 && $sp >= 3));
 
+    # check LTSS activation on host
+    my $status = q(set -o pipefail; SUSEConnect -s | jq -Mr '.[] | select(.identifier == "SLES-LTSS") | .subscription_status');
```

Comment:
```suggestion
    my $ltss_command = q(set -o pipefail; SUSEConnect -s | jq -Mr '.[] | select(.identifier == "SLES-LTSS") | .subscription_status');
```

Please don't do `set -o pipefail`. This will affect all future test runs as well, as this is set globally. I also don't think it's needed, because if any of the commands fail, you will not get the `ACTIVE` string that you're trying to grep.

I would also rename this variable. What you set is a command, not a status.

Reply:
Pipefail I added for debugging purpose, I think it is always useful to identify what is the first failing command in any pipeline. But if you prefer it differently, I remove it.

Reply:
For your understanding: It might be helpful HERE, but it changes the global behaviour of bash for all other test runs as well.

One must look also at the bigger picture and not introduce unintended side-effects for other modules, which then might blow up without a trace.

***

In such cases it's a good practice to either reset the environment after, or to isolate those settings into their own bash scripts, so that the global profile remains clean.